iree/vm/module.h

```diff
@@ -349,6 +389,11 @@ typedef struct iree_vm_module_t {
       iree_host_size_t ordinal, const iree_vm_function_t* function,
       const iree_vm_function_signature_t* signature);
 
+  // Notifies the module of a system signal.
+  iree_status_t(IREE_API_PTR* notify)(void* self,
+                                      iree_vm_module_state_t* module_state,
+                                      iree_vm_signal_t signal);
```

Comment:
Does the TensorFlow Lite C API have some similar concept we could implement now?

Doesn't seem like it? https://github.com/google/iree/blob/main/bindings/tflite/include/tensorflow/lite/c/c_api.h